cluster-operator/src/test/java/io/strimzi/operator/cluster/FeatureGatesTest.java

```diff
@@ -18,8 +18,13 @@
     @ParallelTest
     public void testFeatureGates() {
```

Comment:
I think we should add a method to `FeatureGates` (if it doesn't already have it) to get the known gates, and an assertion here for the known gates. That way it's impossible to add a gate and not update this test.